pilot/pkg/model/push_context.go

```diff
@@ -910,8 +873,8 @@ func (ps *PushContext) updateContext(
 	var servicesChanged, virtualServicesChanged, destinationRulesChanged, gatewayChanged,
 		authnChanged, authzChanged, envoyFiltersChanged, sidecarsChanged, quotasChanged bool
 
-	for k := range pushReq.ConfigsUpdated {
-		switch k {
+	for conf := range pushReq.ConfigsUpdated {
+		switch conf.Kind {
```

Comment:
nit: User the `ServiceEntryKind` , `VirtualServiceKind` defined above in case

Reply:
done